ckanext/cioos_theme/templates/header.html

```diff
@@ -40,6 +40,7 @@ <h4 class="invis">Mobile Navigation</h4>
   <!-- Top Bar -->
   <header id="topmenu" class="nav main-nav">
     <div class="container">
+      {% block header_topmenu %}
       <nav>
         <ul>
           <li><a href="http://twitter.com/cioos_siooc" tabindex="0" aria-label="Twitter"><i class="fa fa-twitter"></i></a></li>
```

Comment:
This line is modified in #29 to allow overriding the socials list. I think you said you wanted to override the logo as well?

Reply:
here is the content of my header.html file:
{% ckan_extends %}

{% block header_logo %}
<a rel="home" href="https://ogsl.ca"><img src="{{ logo_url }}" alt="Logo"></a>
{% endblock %}

{% block header_topmenu %}
<a href="http://cioos.ca"><img width="392" height="41" src="{{h.url_for_static('/siooc_banner_fr.png')}}"></a>
<nav>
  <ul>
    <li><a href="https://www.linkedin.com/company/observatoire-global-du-saint-laurent---ogsl" tabindex="0" aria-label="LinkedIn"><i class="fa fa-linkedin"></i></a></li>
    <li><a href="https://twitter.com/ogsl_slgo?lang=fr" tabindex="0" aria-label="Twitter"><i class="fa fa-twitter"></i></a></li>
    <li><a href="https://www.facebook.com/ogsl.slgo/" tabindex="0" aria-label="Facebook"><i class="fa fa-facebook-square"></i></a></li>
    <li>
      {%- if h.lang() == 'en' -%}
        <a href="{% url_for h.current_url(), locale='fr' %}" tabindex="0" aria-label="French/English">FR</a>
      {%- else -%}
        <a href="{% url_for h.current_url(), locale='en' %}" tabindex="0" aria-label="French/English">EN</a>
      {%- endif -%}
    </li>
    <li >Association régionale du Saint-Laurent</li>
  </ul>
</nav>
{% endblock %}

Reply:
If you prefer, you can merge #29 first and i ll update this PR accordingly